### PR TITLE
Housekeeping related to taking ownership

### PR DESCRIPTION
**Pull request by punkle**
- moves vouch chart under charts/vouch
- create on pull request workflow action with basic linting
- creates README.md
- change contributors list

### CODE REVIEW

File: charts/vouch/Chart.yaml
Comment (martina-if) [inline charts/vouch/Chart.yaml:21]: perhaps keep gavin here?

File: README.md
Comment (martina-if) [inline README.md:11]: reminder to fix this

File: .github/workflows/on-push.yaml
Comment (martina-if) [inline .github/workflows/on-push.yaml:13]: reminder for this

File: .github/workflows/on-pull-request.yaml
Comment (martina-if) [inline .github/workflows/on-pull-request.yaml:30]: should it be just `rm -f tmp.yaml`.

**Review by martina-if — COMMENTED** [review]
Looks good to me. I'll let others pitch in

File: .github/workflows/on-pull-request.yaml
Comment (punkle) [reply .github/workflows/on-pull-request.yaml:30]: yeah I think so.

File: README.md
Comment (punkle) [reply README.md:11]: Need to work with @bnfinet on maybe setting up a github pages place to publish the packages.

File: .github/workflows/on-push.yaml
Comment (punkle) [reply .github/workflows/on-push.yaml]: Done. Kinda.. It's now setup to publish to the github artifacts. As mentioned in the previous comment, we need to work with @bnfinet to get github pages setup.

File: .github/workflows/on-pull-request.yaml
Comment (punkle) [reply .github/workflows/on-pull-request.yaml]: done.

**Comment by punkle** [comment]
@bnfinet The pull request workflow does not appear to be kicking off. Would you mind checking if that is something that you can affect?

**Comment by punkle** [comment]
I think what we need is a github pages setup on this repo, configured against a new branch called "gh-pages". See here for context https://github.com/helm/chart-releaser-action.

**Comment by bnfinet** [comment]
@punkle I think you should have a pending membership request to join the team @vouch/helm-charts 

I'd hope that gives you enough privilege to manage the repo.  Though of course I'm happy to help.

**Comment by bnfinet** [comment]
@punkle looks like @vouch/helm-charts wasn't setup there, my apologies.  I've given that group 'admin' privileges on the repo

File: README.md
Comment (punkle) [reply README.md]: Done now.

**Comment by punkle** [comment]
Hey, I believe this is ready to be merged. 

I guess we should settle out on the etiquette around pull requests. In the meantime, I will wait for at least one approval of the most recent commit before merging changes.

All going to plan the merge action will publish the helm chart to: `https://vouch.github.io/helm-charts/`. We will be very lucky if this works first time.

**Comment by bnfinet** [comment]
Unfortunately I'm not really in a position to test or review.  I'm not currently running a k8s instance and my kube-fu is limited at best.

**Review by bnfinet — APPROVED** [review]
The docs look really great and most everything else LGTM, with the caveat I wrote in the conversation :)

**Comment by punkle** [comment]
Thanks @bnfinet

**Pull request merged**
